### ISSUE TITLE
Python34 and issues for Nuitka-0.6.3

### ISSUE THREAD

**Issue by ManPython** [Nuitka/Nuitka#343]
I made many multiple tests and want provide some issues for Nuitka-0.6.3 with Python34
Hope that can help to improve nuitka.
The python code is just print("Hello!")

1. 
python -m nuitka --mingw --follow-impor
ts --show-progress --output-dir=build\--mingw--follow-imports main.pyw

> Nuitka:INFO:PASS 1:
> Nuitka:INFO:Initial optimization pass.
> Nuitka:INFO:Optimizing module '__main__', 0 more modules to go after that.
> Nuitka:INFO:Doing module local optimizations for '__main__'.
> Nuitka:INFO:Next global optimization pass.
> Nuitka:INFO:Optimizing module '__main__', 0 more modules to go after that.
> Nuitka:INFO:Doing module local optimizations for '__main__'.
> Nuitka:INFO:PASS 2 ... :
> Nuitka:INFO:Next global optimization pass.
> Nuitka:INFO:Optimizing module '__main__', 0 more modules to go after that.
> Nuitka:INFO:Doing module local optimizations for '__main__'.
> Nuitka:INFO:Total memory usage before running scons: 23.61 MB (24756224 bytes):
> 
> cc1.exe: out of memory allocating 65536 bytes
> scons: *** [build\--mingw--follow-imports\main.build\CompiledFunctionType.o] Err
> or 1

2. 
python -m nuitka --mingw --recurse-all
--output-dir=build\--mingw--recurse-all main.pyw

> cc1.exe: out of memory allocating 254624 bytes
> scons: *** [build\--mingw--recurse-all\main.build\MainProgram.o] Error 1cc1.exe: out of 

> memory allocating 312320 bytes
> 
> ../mingw-w64/x86_64-8.1.0-posix-seh-rt_v6-rev0/mingw6
> 4/bin/../lib/gcc/x86_64-w64-mingw32/8.1.0/../../../../x86_64-w64-mingw32/bin/as.
> exe: out of memory allocating 524411 bytes
> scons: *** [build\--mingw--recurse-all\main.build\CompiledFunctionType.o] Error

3. 
python -m nuitka --mingw --standalone -
-output-dir=build\--mingw--standalone main.pyw

> cc1.exe: out of memory allocating 107632 bytes
> scons: *** [build\--mingw--standalone\main.build\MainProgram.o] Error 1

4.  
python -m nuitka --mingw --standalone -
-recurse-all --output-dir=build\--mingw--standalone--recurse-all main.pyw

> cc1.exe: out of memory allocating 71068 bytes
> scons: *** [build\--mingw--standalone--recurse-all\main.build\MainProgram.o] Err
> or 1

**Comment by kayhayen** [comment]
This is bizarrely little. Seeing that it appears to be a 64 bits platform, I am deeply confused.  Also I have used MinGW64 with 8.1 before, so I am not getting it. I am still using 7.3 in my tests. I think I only did the 8 based variant, so checkout if there was a performance increase, which there was none.

I want to claim it's invalid, only because you are alone with this kind of issue, or so I am hoping at least, can you try 0.6.2 maybe?

**Comment by ManPython** [comment]
I made this procedures again and I think that was some mistake here in procedures (many .bat files, commands in notes - I grab logs rapidly to the other notepad to compare and note some issues), but need prevention here. 
If we use:
`--mingw` should be used for `py-x86`
`--mingw64` should be used for `py-x64`
In other case we can call `--mingw` for `py-x64` that can made this mistake and no information about conflit.

**Comment by kayhayen** [comment]
No, actually the MinGW64 should automatically be matched. However, nobody has taken on the issue #225 yet, which would be really helpful. Are you saying that is what happened to you?

**Comment by ManPython** [comment]
OK, I  repeated the tests from the points 1,2,34, and are pass as ok in case of this same info in point 1.
Now understand the source of problem I think.

In my process of tests as reported here I made compialtion for some huge project (QT) and was some problems, long time compilation.
So I start test small scripts as print.
1) In case of test huge project the Fire Fox reported problem with memory by window, I ignoring this as some bug, but after some time the FF i crashed (it was some bug 6mnts ago that I observed this in FF as bug).
2) After this crash when I do test as 1,2,3,4 errors was repeated. 
3) So I closed FF and this same. 
4) After rebot OS and repeating steps this same. But in case where FF was not runed and made again test all was ok. 

Soo.. it's not the problem x86 x64.

Must analise the case of huge project, eg. cut sout some part to see how I can operate with this.
BTW ppls talking about long compilations in cpp and thats way they like Go Lang (however playing there with QT is not nice).
